contracts/modules/LoanOpenings/LoanOpenings.sol

```diff
@@ -89,6 +94,79 @@ contract LoanOpenings is State, LoanOpeningsEvents, VaultController, InterestHan
         );
     }
 
+    function getDefaultLoanParams(
+        address loanToken,
+        address collateralToken,
+        bool isTorqueLoan)
+        public
+        view
+        returns(LoanParams memory loanParamsLocal) 
+    {
+        
+        bytes32 loanParamsId = getPoolLoanParamId(loanToken, collateralToken, isTorqueLoan);
+
+        loanParamsLocal = loanParams[loanParamsId];
+        if (loanParamsLocal.id == 0) {
+            if (isTorqueLoan) {
```

Comment:
do require(isTorqueLoan, "LoanOpenings: Unsupported margin trade token"); here instead of an if statement. This will properly throw an error instead of providing a blank loan params

Reply:
this is view function. if loanParamsLocal.id is zero at the end it will fail opening borrow on 
function borrowOrTradeFromPool: line 69

Reply:
we might aslo support marginTrade with iToken collateral

Reply:
It will fail later on, but failing earlier is just cleaner for knowing what went wrong. As for supporting iToken as collateral for marginTrade, that will likely come later anyways so we can just re-deploy this contract

Reply:
view functions cant' revert

Reply:
they can revert if an error occurs but was more so thinking of borrowOrTradeFromPool that calls this

Reply:
I don't wanna throw errors because I want this function to return defaults

Reply:
in the future for margin we are going to fill it in as well.